docs/transformers/text-normalizer.md

```diff
@@ -17,5 +19,5 @@ This transformer does not have any additional methods.
 ```php
 use Rubix\ML\Transformers\TextNormalizer;
 
-$transformer = new TextNormalizer(true);
+$transformer = new TextNormalizer();
```

Comment:
Nice catch ... we used to allow the user to turn on/off removing extra whitespace